server/core/src/utils.ts

```diff
@@ -15,6 +16,7 @@ import {
 export function createCacheFindAll (storage: ServerStorage): ServerStorage['findAll'] {
   // We will cache all queries for same objects for all derived data checks.
   const queryCache = new Map<string, FindResult<Doc>>()
+  const hierarchy = new Hierarchy()
```

Comment:
It will not work, you have hierarchy in `storage.hierarchy`

Reply:
Fixed, thank you